### ISSUE TITLE
Literature landing page changes

### ISSUE THREAD

**Issue by jmartinm** [inspirehep/inspire-next#581]
We are currently showing the same search tips we currently do in production in the Literature landing page.

![screen shot 2015-11-30 at 11 35 46 am](https://cloud.githubusercontent.com/assets/580727/11469416/82880a20-9756-11e5-8106-35b70ab3f47d.png)

We need to:

* [ ] Decide whether we keep a single syntax advertised in the front page. That has been the external advise given to INSPIRE. Logs reflect that the majority of users either don't use any syntax or use Spires syntax. So if we go for this, advertising only Spires syntax ( maybe rename it to INSPIRE syntax? ) can make sense. We would have to see that Spires syntax supports all the common use cases.
* [ ] Test that the searches advertised work.
* [ ] Update link `More search tips and full help` to point to https://inspirehep.net/info/hep/search-tips (and remove the `full help` part? This link to search tips can also be moved close to the search bar and always keep it there.
* [ ] At the moment, the search examples under the search bar are redundant with the `How to Search` part.
* [ ] As the Literature landing page becomes the landing page of INSPIRE Labs we might want to add some more content ?

**Comment by nikpap** [comment]
None of the searches advertised in the `example queries` or `how to search` box currently work. These will need to be tweak appropriately. In addition, searches using `and` or `or` produce a 500 server error.

Some notes regarding search:

- [ ] `find a` only works with author full name. For example `find a Maschke` will produce 0 results, while `find a Maschke, a.W.` will work. In the production website, writing the surname of the author seems to be enough to produce results.
- [ ] Searching for an author's surname produces 0 results.
- [ ] Searching for an e-Print produces wrong results. A search for `hep-ph/9607327` will display as a result the record with e-Print `hep-ph/9610500`, despite the fact that there is another record with exactly the search query we're looking for (recid: 420758)
- [ ] Searching for `arXiv:9708453` produces no results - should show recid: 362983

**Comment by bing13** [comment]
I think it makes sense to advertise on this page only free form and "INSPIRE syntax".  We can offer Invenio/Elastic syntax from a help page for special applications, etc.  Offering another syntax might be helpful to users who already know that alternate syntax, and ourselves, but those are small numbers.

**Comment by jmartinm** [comment]
> Searching for arXiv:9708453 produces no results - should show recid: 362983

This gives no results on production either https://inspirehep.net/search?ln=en&p=arXiv%3A9708453&of=hb&action_search=Search&sf=earliestdate&so=d

**Comment by kaplun** [comment]
> Searching for an e-Print produces wrong results. A search for hep-ph/9607327 will display as a result the record with e-Print hep-ph/9610500, despite the fact that there is another record with exactly the search query we're looking for (recid: 420758) 

Yep, this is due to the fact the current global search is adding references, but is not adding arxiv/reportnumbers... (the wrong record is matched because of the references).

**Comment by jmartinm** [comment]
@jacquerie The search log analysis you are doing should show the current strengths and weaknesses of the search in Labs. i think based on the output we can adapt the search queries we advertise in the front page.
